### PR TITLE
finish version control

### PR DESCRIPTION
**Pull request by fqliao**

### CODE REVIEW

File: libprecompiled/extension/WedprPrecompiled.cpp [hunk 1]
```diff
@@ -41,6 +41,12 @@ using namespace dev::storage;
 using namespace dev::precompiled;
 
 // confidential payment
+const char CONFIDENTIAL_PAYMENT_VERSION[] = "v0.2-generic";
+const string CONFIDENTIAL_PAYMENT_REGEX_WHITELIST = "^v0.2-generic$";
```
Comment (qyan-dev) [inline libprecompiled/extension/WedprPrecompiled.cpp]: Is it possible to use const char CONFIDENTIAL_PAYMENT_REGEX_WHITELIST[] ?

File: libprecompiled/extension/WedprPrecompiled.h [hunk 2]
```diff
@@ -80,10 +99,16 @@ class WedprPrecompiled : public dev::blockverifier::Precompiled
...
 
+    bytes anonymousAuctionIsCompatible(dev::eth::ContractABI& abi, bytesConstRef& data);
+    bytes anonymousAuctionGetVersion(dev::eth::ContractABI& abi);
     bytes verifyBidSignatureFromBidRequest(dev::eth::ContractABI& abi, bytesConstRef& data);
     bytes VerifyBidSignatureFromBidComparisonRequest(
         dev::eth::ContractABI& abi, bytesConstRef& data);
     bytes verifyWinner(dev::eth::ContractABI& abi, bytesConstRef& data);
+
+private:
+    int isCompatible(const std::string& targetVersion, const std::string& regexWhitelist,
```
Comment (qyan-dev) [inline libprecompiled/extension/WedprPrecompiled.h]: const std::string& -> const char* ?

File: libprecompiled/extension/WedprPrecompiled.cpp (hunk 1 above)
Comment (fqliao) [reply libprecompiled/extension/WedprPrecompiled.cpp]: done

File: libprecompiled/extension/WedprPrecompiled.h (hunk 2 above)
Comment (fqliao) [reply libprecompiled/extension/WedprPrecompiled.h]: done

**Pull request merged**